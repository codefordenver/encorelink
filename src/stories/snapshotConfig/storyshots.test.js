import initStoryshots from '@storybook/addon-storyshots';
import moment from 'moment-timezone';
import * as addonInfo from '@storybook/addon-info';

moment.tz.setDefault('Europe/London');
moment.tz.guess = jest.fn(() => 'Europe/London');
addonInfo.withInfo = jest.fn(() => (renderer) => renderer);

process.env.REACT_APP_GOOGLE_API_KEY = 'REDACTED';

initStoryshots();
