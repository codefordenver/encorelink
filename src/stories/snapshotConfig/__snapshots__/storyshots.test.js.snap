// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Event Event with customizable props 1`] = `
<div
  className="row"
>
  <div
    className="small-12 columns"
  >
    <h3>
      The best event
    </h3>
    <h4>
      Tuesday, January 1, 1901
       | 
      11:00-2:30 am GMT
    </h4>
    <p>
      Galvanize, Golden Triangle, Denver
    </p>
  </div>
  <div
    className="small-12 medium-8 large-6 columns"
  >
    <div
      className="flex-video"
    >
      <iframe
        allowFullScreen={true}
        frameBorder="0"
        height="350"
        src="https://www.google.com/maps/embed/v1/place?key=REDACTED&q=Galvanize%2C%20Golden%20Triangle%2C%20Denver"
        style={
          Object {
            "border": 0,
          }
        }
        title="Location Map"
        width="600"
      />
    </div>
  </div>
  <div
    className="small-12 columns"
  >
    <div>
      <h5>
        Additional Information
      </h5>
      <p>
        These are the notes on the event
      </p>
    </div>
  </div>
  <div
    className="small-12 columns"
  >
    <p>
       
      <a
        className="button secondary"
        href="mailto:thedude@loves.me"
      >
        Contact Organizer
      </a>
       
      <a
        className="button secondary"
        onClick={[Function]}
        style={Object {}}
      >
        Go Back
      </a>
    </p>
  </div>
</div>
`;

exports[`Storyshots EventRow Event row with customizable props 1`] = `
<div
  className="row event-row"
>
  <div
    className="small-12 medium-2 columns"
  >
    Tuesday, January 2, 2018
  </div>
  <div
    className="small-12 medium-2 columns"
  >
    2:00-4:00 am GMT
  </div>
  <div
    className="small-12 medium-5 columns"
  >
    music
     
    denver
  </div>
  <div
    className="small-12 medium-1 columns"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      details
    </a>
  </div>
  <div
    className="small-12 medium-2 columns"
  >
    <div
      className="react-add-to-calendar"
    >
      <div
        className="react-add-to-calendar__wrapper"
      >
        <a
          className="react-add-to-calendar__button"
          onClick={[Function]}
        >
          <span>
            add to calendar 
            <i
              className="react-add-to-calendar__icon--right fa fa-caret-down"
            />
          </span>
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Header Logged In: Organizer 1`] = `
<header
  className="top-bar"
>
  <div
    className="top-bar-left"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="EncoreLink"
        className="logo-img"
        src="encorelink-logo.png"
      />
    </a>
  </div>
  <div
    className="top-bar-right text-right"
  >
    <div>
      Hello, 
      <a
        onClick={[Function]}
        style={Object {}}
      >
        organizer@encorelink.com
      </a>
       
       
      <button
        className="button"
        onClick={[Function]}
      >
        Log out
      </button>
    </div>
  </div>
</header>
`;

exports[`Storyshots Header Logged in: Musician 1`] = `
<header
  className="top-bar"
>
  <div
    className="top-bar-left"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="EncoreLink"
        className="logo-img"
        src="encorelink-logo.png"
      />
    </a>
  </div>
  <div
    className="top-bar-right text-right"
  >
    <div>
      Hello, 
      <a
        onClick={[Function]}
        style={Object {}}
      >
        musician@encorelink.com
      </a>
       
      <a
        className="button"
        onClick={[Function]}
        style={Object {}}
      >
        Dashboard
      </a>
       
      <button
        className="button"
        onClick={[Function]}
      >
        Log out
      </button>
    </div>
  </div>
</header>
`;

exports[`Storyshots Header Not logged in 1`] = `
<header
  className="top-bar"
>
  <div
    className="top-bar-left"
  >
    <a
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="EncoreLink"
        className="logo-img"
        src="encorelink-logo.png"
      />
    </a>
  </div>
  <div
    className="top-bar-right text-right"
  >
    <span>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        About Us
      </a>
      <a
        className="button login"
        onClick={[Function]}
        style={Object {}}
      >
        Log in
      </a>
    </span>
  </div>
</header>
`;
